src/game.cpp

```diff
@@ -10848,6 +10848,10 @@ void game::butcher()
         add_msg(m_info, _("You can't butcher while driving!"));
         return;
     }
+    if( !m.can_put_items_ter_furn( u.pos() ) ) {
+        add_msg( m_info, _( "You can't butcher while standing here!" ) );
```

Comment:
The message should probably say that "you can not access the items" (that's what prevents the butchering) when there are items (visible to the player), but they are sealed away.

The situation for the rubble terrain is slightly different. The player can't see the items and may not know of their existence. It should yield the same message as when attempting to butcher on an empty tile. From the players knowledge it *is* an empty tile.